eng/scripts/Create-APIView.ps1

```diff
@@ -0,0 +1,326 @@
+  }
+
+  git checkout $currentBranch
+
+  LogGroupStart " See all generated Swagger APIView Artifacts..."
+  Get-ChildItem -Path $swaggerAPIViewArtifactsDirectory -Recurse
+  LogGroupEnd
+}
+
+<#
+.DESCRIPTION
+  Create APIView for the published packages. Send DevOps artifacts information to APIView to create APIView for the published packages.
+
+.PARAMETER ArtiFactsStagingDirectory
+  The DevOps artifacts staging directory. Use $(Build.ArtifactStagingDirectory) on DevOps
+.PARAMETER APIViewArtifactsDirectoryName
+ Temporary Directory for processing the APIView artifacts
+.PARAMETER APIViewArtifactsName
+  The name of the APIView artifact
+.PARAMETER APIViewUri
+ The EndPoint for creating APIView https://apiviewstagingtest.com/PullRequest/DetectAPIChanges
+.PARAMETER BuildId
+TGhe BuildId of the Run
+.PARAMETER RepoName
+  Repo name eg Azure/azure-rest-api-specs
+.PARAMETER PullRequestNumber
+  The PR number
+.PARAMETER Language
+  The language of the resource provider `Swagger`
+.PARAMETER CommitSha
+  The commit sha of the current branch. Uusally the merge commit of the PR.
+#>
+function New-RestSpecsAPIViewReviews {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$ArtiFactsStagingDirectory,
+    [Parameter(Mandatory = $true)]
+    [string]$APIViewArtifactsDirectoryName,
+    [Parameter(Mandatory = $true)]
+    [string]$APIViewArtifactsName,
+    [Parameter(Mandatory = $true)]
+    [string]$APIViewUri,
+    [Parameter(Mandatory = $true)]
+    [string]$BuildId,
+    [Parameter(Mandatory = $true)]
+    [string]$RepoName,
+    [Parameter(Mandatory = $true)]
+    [string]$PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string]$Language,
+    [Parameter(Mandatory = $true)]
+    [string]$CommitSha
+  )
+
+  $apiViewArtifactsDirectory = [System.IO.Path]::Combine($ArtiFactsStagingDirectory, $APIViewArtifactsDirectoryName)
+  $publishedPackages = Get-ChildItem -Path $apiViewArtifactsDirectory -Directory -ErrorAction SilentlyContinue
+
+  Write-Host "Published packages: $publishedPackages"
+
+  $createAPIViewFailed = $false
+
+  $publishedPackages | ForEach-Object {
+    $apiViewArtifacts = Get-ChildItem -Path $_.FullName -File -Filter "*.json" -ErrorAction SilentlyContinue
+    $query = [System.Web.HttpUtility]::ParseQueryString('')
+
+    $apiViewArtifacts | ForEach-Object {
+      if ($_.BaseName.EndsWith("New")) {
+        $query.Add("codeFile", $_.Name)
+      }
+      elseif ($_.BaseName.EndsWith("Baseline")) {
+        $query.Add("baselineCodeFile", $_.Name)
+      }
+    }
+
+    if (-not $query['codeFile']) {
+      LogWarning "'New' APIView token file not found for resource provider '$($_.BaseName)'. Skipping APIView creation."
+      return
+    }
+
+    if (-not $query['baselineCodeFile']) {
+      LogWarning "'Baseline' APIView token file not found for resource provider '$($_.BaseName)'. Skipping APIView creation."
```

Comment:
This will be an issue for a new service. Currently new service is not supposed to use Swagger for specs so chances are slim to run into this issue for Swagger. But TypeSpec should definitely support APIView even if there is no baseline for that resource provider (RP) to create a revision for a brand new RP.

Reply:
It's still better to remove the `return` here to support the usecase instead of skipping to create API review when baseline is not available.